lib/Exads/Api/Login.php

```diff
@@ -24,4 +24,18 @@ public function getToken($username, $password)
 
         return $this->post($this->getPath(), $data);
     }
+
+    /**
+     * @param string $api_token
+     *
+     * @return string
+     */
+    public function getTokenByApiToken($api_token)
```

Comment:
how about renaming getToken($username, $password) -> getToken($usernameOrToken, $password = null) and test in there the array we want to build?